internal/gittuf-git/pull/pull.go

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package pull
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/gittuf/gittuf/internal/gittuf-git/args"
+)
+
+func GittufPull(gitArgs args.Args) error {
+	// 0. Determine appropriate env vars
+	// 1. Pull non-RSL changes
+	// 2. Pull RSL changes
+	// 3. Pull policy changes
+
+	var env = os.Environ()
+
+	// 0
+	if gitArgs.ChdirIdx > 0 {
+		env = append(env, "GIT_DIR="+gitArgs.GlobalFlags[gitArgs.ChdirIdx])
+	}
+
+	// 1
+	cmdArgs := []string{gitArgs.Command}
+	cmdArgs = append(cmdArgs, gitArgs.Parameters...)
+	gitPullCmd := exec.Command("git", cmdArgs...)
```

Comment:
What happens if there's a symlink `~/.bin/git -> /usr/bin/gittuf-git`? We may not be able to guarantee this will use the correct `git` all the time, but perhaps we can do better. For now, a TODO is probably ok.